homeassistant/components/sftp_storage/strings.json

```
@@ -0,0 +1,37 @@
+{
+  "config": {
+    "step": {
+      "user": {
+        "description": "Set up remote SFTP Storage",
```

If we treat "SFTP Storage" as the integration name the word "remote" needs to be dropped here.
Otherwise we get a translation mismatch.

```suggestion
        "description": "Set up SFTP Storage",
```